orchagent/notifications.cpp

```diff
@@ -30,5 +34,9 @@ void on_switch_shutdown_request(sai_object_id_t switch_id)
     /* TODO: Later a better restart story will be told here */
     SWSS_LOG_ERROR("Syncd stopped");
 
-    exit(EXIT_FAILURE);
+#ifdef ASAN_ENABLED
+    __lsan_do_leak_check();
+#endif
+
```

Comment:
Please add a comment why we replace exit with quick_exit

Reply:
Done